Solutions/GoogleCloudPlatformDNS/Analytic Rules/GCPDNSMultipleErrorsQuery.yaml

```diff
@@ -0,0 +1,36 @@
+id: da04a5d6-e2be-4cba-8cdb-a3f2efa87e9e
+name: Google DNS - Multiple errors to same domain
+description: |
+  'Detects multiple errors to same domain.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: GCPDNSDataConnector
+    dataTypes:
+      - imDNSGCPCloudDNS
+queryFrequency: 1h
+queryPeriod: 1h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - CommandAndControl
+relevantTechniques:
+  - T1095
+query: |
+  let threshold = 2;
+  let err = dynamic(['NXDOMAIN', 'SERVFAIL', 'REFUSED']);
+  imDNSGCPCloudDNS
+  | where EventResultDetail in~ (err)

```

Comment:
Since you are using the parser in queries, you will have to change field name EventResultDetail to EventResultDetails.
Only then the validations would pass. Thank You.